microsoft-service-bus/2.2/modules/ROOT/pages/index.adoc

```diff
@@ -259,18 +282,18 @@ To enable the SSL checks, follow these steps to import the certificate:
 . Use the PowerShell cmdlet https://msdn.microsoft.com/library/azure/jj248762%28v=azure.10%29.aspx[Get-SBAutoGeneratedCA] to download the certificate locally in the machine running the Windows Service Bus. +
 For this example, the certificate file is exported to `%temp%\AutoGeneratedCA.cer`.
 . Running as Administrator, go to `%programfiles%\Java\jre7`. +
-you must be running as Administrator to perform a certificate import with `Keytool.exe`. 
-. Verify that the `bin\keytool.exe` tool and `lib\security\cacerts` exists. 
+you must be running as Administrator to perform a certificate import with `Keytool.exe`.
+. Verify that the `bin\keytool.exe` tool and `lib\security\cacerts` exists.
 . Enter the following command: +
 `bin\keytool.exe –list –keystore lib\security\cacerts`
 . Import the autogenerated Service Bus certificate by running the following command: +  `bin\keytool.exe –importcert –alias AppServerGeneratedSBCA –file %temp%\AutoGeneratedCA.cer –keystore lib\security\cacerts –v`
 . Enter the password when prompted (the default is “changeit”).  +
-If you don't know the password, you can't perform the import.  
+If you don't know the password, you can't perform the import.
```

Comment:
Not sure this line about not knowing the password is necessary.

Reply:
I'll remove it.